.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       matrix:
         redis-version: ["~> 4.2", "~> 5"]
         ruby-version: ['2.7', '3.0', '3.1', '3.2', '3.3', '3.4']
-        sidekiq-version: ['~> 6', '~> 7']
+        sidekiq-version: ['~> 7.3']
```

Comment:
Sidekiq 8 only supports Ruby 3.2 and later, so this will be added probably once support for 8 is truly started:

```yaml
        exclude:
          - sidekiq-version: '~> 8'
            ruby-version: '2.7'
          - sidekiq-version: '~> 8'
            ruby-version: '3.0'
          - sidekiq-version: '~> 8'
            ruby-version: '3.1'
```